astro/src/content/blog/fusionauth-reinvent-2024.mdx

```diff
@@ -0,0 +1,64 @@
+---
+publish_date: 2024-11-15 
+title: "FusionAuth at AWS re:Invent 2024"
+description: Join us at AWS re:Invent 2024. Meet the team, get demos, and win some great prizes.
+authors: Dan Stotts
+image: /img/blogs/reinvent-2024/re-invent-2024.png
+categories: Events
+tags: reinvent, 2024 events, fusionauth
+excerpt_separator: "{/* more */}"
+---
+
+We’re thrilled to announce that FusionAuth will be at [AWS re:Invent 2024](https://reinvent.awsevents.com/), and we can’t wait to connect with you! We’ve planned an array of activities and opportunities for engagement throughout the event.
```

Comment:
In the past we've also highlighted some interesting sessions. See https://fusionauth.io/blog/fusionauth-re-invent-2023

Don't know if we want to do that here. Happy to provide those if you want.

Reply:
If you have them, please do send some my way. I'm just posting what Dan Stotts sent to me, but I'm happy to add them.

Reply:
Here are four I'm interested in attending. You could say something like: "we're of course interested in some of the sessions too. Here are four that some team members found especially interesting"

SEC346: Customer insights on application authorization 
SAS312: Next-generation SaaS tenant isolation strategies 
DAT335: Safer, simpler, faster upgrades with Amazon RDS Blue/Green Deployments 
KUB323: Observability strategies for Amazon EKS workloads